package.xml

```diff
@@ -12,37 +12,19 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roscpp</build_depend>
-  <run_depend>roscpp</run_depend>
-  <build_depend>rospy</build_depend>
-  <run_depend>rospy</run_depend>
-
-  <build_depend>tf</build_depend>
-  <run_depend>tf</run_depend>
-
-  <build_depend>cv_bridge</build_depend>
-  <run_depend>cv_bridge</run_depend>
-
-  <build_depend>pcl_conversions</build_depend>
-  <run_depend>pcl_conversions</run_depend>
-
-  <build_depend>std_msgs</build_depend>
-  <run_depend>std_msgs</run_depend>
-  <build_depend>sensor_msgs</build_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <run_depend>geometry_msgs</run_depend>
-  <build_depend>nav_msgs</build_depend>
-  <run_depend>nav_msgs</run_depend>
-  <build_depend>visualization_msgs</build_depend>
-  <run_depend>visualization_msgs</run_depend>
-
   <build_depend>message_generation</build_depend>
-  <run_depend>message_generation</run_depend>
-  <build_depend>message_runtime</build_depend>
-  <run_depend>message_runtime</run_depend>
-
-  <build_depend>GTSAM</build_depend>
-  <run_depend>GTSAM</run_depend>
+  <depend>GTSAM</depend>
```

Comment:
I have seen `GTSAM not found` for several times. Probably it should be `gtsam`.
Also then you can remove gtsam from common-apt.

Reply:
The `libgtsam` is not registered as a ROS package, but a CMake library. so the dependency has no meaning.
We can just remove it.

Reply:
Removed